app/views/manage_assignments/courses/_attachments.html.erb

```diff
@@ -1,12 +1,10 @@
-<table class="assignment-attachments is-hidden">
-  <tbody>
-    <% attachments.each do |attachment| %>
-      <tr>
-        <td><%= attachment.file_file_name %></td>
-        <td><%= link_to t(".delete-link"),
-              manage_assignments_attachment_path(attachment),
-              method: :delete %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+<ul class="assignment-attachments is-hidden attachments">
```

Comment:
I wonder if these classes should go `attachments assignment-attachments is-hidden`. I realize that's a super nitpick, but I usually like to read things as general to specific. Maybe they are in that order currently?